SimMuon/RPCDigitizer/src/RPCDigiProducer.cc

```diff
@@ -73,6 +73,7 @@ void RPCDigiProducer::beginRun(const edm::Run& r, const edm::EventSetup& eventSe
    edm::ESHandle<RPCClusterSize> clsRcd;
    eventSetup.get<RPCClusterSizeRcd>().get(clsRcd);
 
+   theRPCSimSetUp->setGeometry( pGeom );
```

Comment:
ok, i have to remove the latter... I overlooked to clean up this file.
it is important that it is done before theRPCSimSetUp->setRPCSetUp(noiseRcd->getVNoise(), clsRcd->getCls());
i ll update the PR.